oshdb/src/main/java/org/heigit/ohsome/oshdb/OSHDBTag.java

```diff
@@ -27,15 +28,15 @@ public boolean isPresentInKeytables() {
 
   @Override
   public boolean equals(Object o) {
-    return o instanceof OSHDBTag &&
-        ((OSHDBTag)o).key == this.key && ((OSHDBTag)o).value == this.value;
+    return o instanceof OSHDBTag 
```

Comment:
superfluous trailing whitespace

Reply:
should be fixed, but we should also add this as a rule to our checkstyle. So that it becomes more visible.